### PR TITLE
add minute.ly test to no-one

### PR DESCRIPTION
**Pull request by jamesgorrie**
# Minute test

This is not meant to go live for users, it's just so we can test [Minute](https://minute.ly)'s script and see what teething problems we might have.
## What is the value of this and can you measure success?

We would know how technically viable it is to work with Minute
## Does this affect other platforms - Amp, Apps, etc?

Nope
## Screenshots

:santa: 
## Request for comment

@akash1810 @sndrs 
## 

_Does this PR meet the [contributing guidelines](https://github.com/guardian/frontend/blob/issue_pr_templates/.github/CONTRIBUTING.md#submission)?_

### CODE REVIEW

File: static/src/javascripts/projects/common/modules/experiments/tests/minute.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+define([
+], function (
+) {
+    return function () {
+        this.id = 'Minute';
+        this.start = '2016-05-11';
+        this.expiry = '2016-04-11';
```
Comment (akash1810) [inline static/src/javascripts/projects/common/modules/experiments/tests/minute.js]: `expiry` should be > `start`?

Comment (jamesgorrie) [inline static/src/javascripts/projects/common/modules/experiments/tests/minute.js]: Thanks - changed.

File: static/src/javascripts/bootstraps/enhanced/minute.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
+define([], function () {
+    window._min =  {_publisher: 'MIN-21000'};
```
Comment (sndrs) [inline static/src/javascripts/bootstraps/enhanced/minute.js]: feels a bit unsafe, is that a minutely thing?

**Comment by sndrs** [comment]
:+1:

File: static/src/javascripts/bootstraps/enhanced/minute.js (hunk 2 above)
Comment (jamesgorrie) [inline static/src/javascripts/bootstraps/enhanced/minute.js]: Yup - it is unsafe.
We have access to their developers, so I will ask them to have some form of constructor that can take this.

**Comment by akash1810** [comment]
:+1:

**Pull request merged**